packages/handlers/odata/src/declarations.d.ts

```diff
@@ -0,0 +1,2 @@
+declare module 'string-interpolation/src';
+declare module 'graphql-fields';
```

Comment:
No typings for this one? And do we really need this external dependency, or we can maybe do it without it?

Reply:
I found one for graphql-fields. It is a nice package that handles inline fragments, `@include` and `@skip` directives as well. We can keep it.